src/components/menu/__internal__/submenu/submenu.component.tsx

```diff
@@ -111,6 +111,9 @@ const Submenu = React.forwardRef<
     const [submenuFocusId, setSubmenuFocusId] = useState<string | null>(null);
     const [submenuItemIds, setSubmenuItemIds] = useState<(string | null)[]>([]);
     const [characterString, setCharacterString] = useState("");
+    const [applyFocusRadius, setApplyFocusRadius] = useState<
```

Comment:
suggestion: I would just make this a boolean type and coerce the value with `!!` when it's set etc